easybuild/easyconfigs/d/DeePMD/DeePMD-0.10.2-CrayGNU-18.08.eb

```diff
@@ -0,0 +1,71 @@
+easyblock = "CmdCp"
+
+name = 'DeePMD'
+version = '0.10.2'
+
+homepage = 'https://github.com/deepmodeling/deepmd-kit/'
+description = """DeePMD-kit is a package written in Python/C++, designed to
+minimize the effort required to build deep learning based model of interatomic
+potential energy and force field and to perform molecular dynamics."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'dynamic': True, 'verbose': False}
+
+builddependencies = [
+    ('CMake', '3.11.4', '', True),
```

Comment:
Please use `CMake/3.12.0`, this is the latest module available on the new software stack on `Piz Daint`